src/Match2Cases.jl

```diff
@@ -56,19 +56,8 @@ function build_state_machine_core(
             succ = successors(code)
             for i in 1:length(succ)
                 push!(work_queue, succ[i])
```

Comment:
```suggestion
            for s in successors(code)
                push!(work_queue, s)
```
possible cleanup since `i` is no longer used

Reply:
`push!(work_queue, successors(code)...)`